modules/monitoring-creating-cluster-monitoring-configmap.adoc

```diff
@@ -3,36 +3,55 @@
 // * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
 
 [id="creating-cluster-monitoring-configmap_{context}"]
-= Creating cluster monitoring ConfigMap
+= Creating a cluster monitoring ConfigMap
 
-To configure the Prometheus Cluster Monitoring stack, you must create the cluster monitoring ConfigMap.
+To configure the {product-title} monitoring stack, you must create a cluster monitoring ConfigMap.
 
 .Prerequisites
 
-* An installed `oc` CLI tool
-* Administrative privileges for the cluster
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
 
 .Procedure
 
-. Check whether the `cluster-monitoring-config` ConfigMap object exists:
+. Check whether a `cluster-monitoring-config` ConfigMap object exists:
 +
+[source,terminal]
 ----
 $ oc -n openshift-monitoring get configmap cluster-monitoring-config
 ----
 
-. If it does not exist, create it:
+. Configure the ConfigMap.
+** If the ConfigMap does not exist:
+.. Create the following YAML manifest. In this example the file is called `cluster-monitoring-config.yaml`:
 +
+[source,yaml]
 ----
-$ oc -n openshift-monitoring create configmap cluster-monitoring-config
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+----
```

Comment:
I think it is useless to create an empty ConfigMap then add the parameter settings, we can add the settings in the ConfigMap and create it if  the ConfigMap does not exist, example
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: cluster-monitoring-config
  namespace: openshift-monitoring
data:
  config.yaml: |
    prometheusK8s:
      retention: 4h
```

Reply:
@juzhao thank you for your comment! The problem is that many of the items in this assembly and others have a prerequisite stating that you have created a ConfigMap. As such, I think it is necessary to have that as a separate module so that we can refer to it in the .Additional resources section. This saves us from having to reiterate this same step each time. I had a BZ where someone was confused by the prerequisite unless it was explicitly referenced. Based on this, my view is that we should have a separate module that focuses on creating the ConfigMap and getting the data section ready, which lays the groundwork for subsequent config modules.

* Please see my (as yet not finished PR) for 4.6 which relies on this heavily: https://github.com/openshift/openshift-docs/pull/24722. 
* Here is the preview from that PR for the 'Configuring the monitoring stack' section: https://osdocs1240_user_workload_monitoring--ocpdocs.netlify.app/openshift-enterprise/latest/monitoring/configuring-the-monitoring-stack.html

Reply:
Agree with @pneedle-rh on this. This section if for creating the configmap. Following steps will instruct how to configure the config map in meaningful way: https://docs.openshift.com/container-platform/4.4/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html#creating-cluster-monitoring-configmap_configuring-monitoring
I would rather just delete the last step from this PR.

Reply:
@olga-mir which step is it specifically that you would rather delete?

Reply:
@pneedle-rh - the last "edit": https://github.com/openshift/openshift-docs/pull/25060/commits/0de7126737ff7b4e588f4a80bf92611aee310efc
For two reasons: we are not instructing actually "edit" anything, this payload is already what the configmap has. 
and second - it is duplication of the configmap payload just a few lines above

Reply:
@olga-mir thank you, that is a good point. Please see my related comment here:

https://github.com/openshift/openshift-docs/pull/25060#discussion_r490025840

Reply:
@pneedle-rh thanks for confirming. I am losing track of what is going on with this PR. I think we now agree to have these lines removed https://github.com/openshift/openshift-docs/pull/25060/files#diff-89302581129a792feee2baace787a6dbR45-R51
the rest seems to align with what you have in your PR.

Reply:
The changes look good to me.